transport/trojan/trojan.go

```diff
@@ -8,12 +8,13 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
-	tlsC "github.com/Dreamacro/clash/component/tls"
 	"io"
 	"net"
 	"net/http"
 	"sync"
 
+	tlsC "github.com/Dreamacro/clash/component/tls"
+
```

Comment:
Duplicate new line. Could you please remove the new line and re-fmt?

Reply:
For sure

Reply:
Fixed